medusa/orchestration.py

```diff
@@ -13,9 +13,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import paramiko
 import logging
-from pssh.clients.miko import ParallelSSHClient
+from pssh.clients.ssh import ParallelSSHClient
```

Comment:
I remember not switching to `.ssh`  because this implementation didn't handle agent forwarding and/or running `sudo` commands correctly 🤔 

I need to give this a good try on a cluster to see how it works.

Reply:
And thanks for submitting a PR @optplx !
I'll report back when I have ran some tests with this code.

Reply:
good point, I have not tested this indeed. Though when bumping pssh, the .`miko` doesn't exist anymore

Reply:
Hello @adejanovski can I help on the tests? If you have specific scenarios in mind I would be happy to have a look